docker-compose.official.yml

```diff
@@ -115,8 +118,7 @@ services:
       - ./dockerfiles/scripts:/tmp/scripts
 
   database:
-    container_name: eoapi.db
-    image: ghcr.io/stac-utils/pgstac:v0.7.10
+    image: ghcr.io/stac-utils/pgstac:v0.8.1
```

Comment:
Updated the `image` to use the latests tags, not the `latest` which can usually have breaking changes